main.py

```diff
@@ -114,40 +119,65 @@ def reply(update, context):
 def group(update, context):
     """Checks for profanity in messages and responds to that."""
 
-    if update.message is not None and update.message.text is not None:
```

Comment:
why

Reply:
applied text filter, so update.message.text should theoretically never be none

Reply:
Okay that makes sense, nice.